src/commands/basic/init.js

```diff
@@ -156,17 +156,40 @@ const fetchTemplate = async (template) => {
     client: false,
   };
 
-  // Show up a suitable prompt whether if the user requires a Full stack application (Express.js)
-  const { requireServer } = await inquirer.prompt({
-    name: 'requireServer',
-    type: 'confirm',
+  const requireServerChoices = ['Yes', 'No'];
+  // Doesn't support more option for GraphQl
+  if (template !== 'GraphQL') {
+    requireServerChoices.push('More');
+  }
+  // Show up a suitable prompt whether if the user requires a Full stack application (Default: Express.js)
+  const { requireServerOption } = await inquirer.prompt({
+    type: 'list',
     message: 'Do you require server side template (Express.js)',
+    name: 'requireServerOption',
+    choices: requireServerChoices,
```

Comment:
Let's keep the `confirm` prompt as it is, and if the user prefers to have a server-side template, ask his/her framework of choice (`Express.js`/`Hapi.js`).

Reply:
I think MEVN will lose it's meaning if the user prefers the server-side template and then again he has to choose a server-side template from options. That's why I made it as a list. If a user prefers a server-side template then by default it should be `express.js` @jamesgeorge007

Reply:
We shouldn't have supported any of those frameworks if that was the case. Please make the suggested change, and we are good to go.

Reply:
Sure :+1: